simvue/run.py

```diff
@@ -435,10 +470,13 @@ def save(self, filename, category, filetype=None, preserve_path=False):
             self._error('Run is not active')
             return False
 
-        if not os.path.isfile(filename):
-            self._error(f"File {filename} does not exist")
+        # check data type, if pickleable object create a file, if file check path
+        try: 
+            filename = get_filename_input(input_data)
```

Comment:
This is incorrect. The function has a second argument:
```
def get_filename_input(input_data, name_tag)
```

Reply:
Yes. Will fix and add a test to flag this.